src/app/src/test/kotlin/org/imperial/mrc/hint/helpers/LogMemoryAppender.kt

```diff
@@ -7,29 +7,27 @@ import java.util.*
 import java.util.function.Predicate
 import java.util.stream.Collectors
 
-class MemoryAppender : ListAppender<ILoggingEvent?>() {
+class LogMemoryAppender : ListAppender<ILoggingEvent?>() {
     fun reset() {
-        list.clear()
+        this.list.clear()
```

Comment:
Not sure it's really useful to use `this` when there isn't any ambiguity about where the list is coming from.

Reply:
I'll revert that, can't remember why I did that!